### PR TITLE
add karma for unit tests

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,22 @@
   "license": "LGPL 2+",
   "main": "dist/image-widgets.js",
   "devDependencies": {
+    "angular": "^1.6.6",
```
Comment (petervo) [inline package.json]: Do we have to pull these in as deps? Or can we load them out of bower like we do now.

**Comment by thomasmckay** [comment]
updated package.json and karma.conf.js to use existing angular installed via bower

**Comment by petervo** [comment]
Added a slightly modified version here. https://github.com/petervo/registry-image-widgets/tree/karma-test

This keeps webpack from picking up the tests and avoids any changes to dist.

**Comment by stefwalter** [comment]
Nice! How do we hook this up for semaphore or Travis? Or is that a next step.

**Pull request merged**